tests/test_search.py

```diff
@@ -667,41 +846,44 @@ def test_search_startswith(model, context, app):
 
     app.authmodel(model, ["search"])
 
-    # single field search
-    resp = app.get(f'/{model}?report_type.startswith("VM")')
+    resp = app.get(f"/{model}?{query}")
     data = resp.json()["_data"]
     assert len(data) == 1
     assert data[0]["_id"] == r2["_id"]
 
-    # single field search, case insensitive
-    resp = app.get(f'/{model}?report_type.lower().startswith("vm")')
-    data = resp.json()["_data"]
-    assert len(data) == 1
-    assert data[0]["_id"] == r2["_id"]
 
-    # multi field search
-    # test if operators are joined with AND logic
-    resp = app.get(f'/{model}?status.startswith("in")&report_type.lower().startswith("vm")')
-    data = resp.json()["_data"]
-    assert len(data) == 1
-    assert data[0]["_id"] == r2["_id"]
+@pytest.mark.models(
+    "backends/mongo/Report",
+    "backends/postgres/Report",
+)
+@pytest.mark.parametrize(
+    "query, valid",
+    [
+        ('status.startswith("valid")', True),
+        ('notes.create_date.startswith("2019-04-20")', False),
+        ('status._sw="valid"', True),
+        ('notes.create_date._sw="2019-04-20"', False),
+    ],
+)
+def test_search_startswith_valid(model, context, app, query, valid):
+    (
+        r1,
+        r2,
+        r3,
+    ) = _push_test_data(app, model)
```

Comment:
Can't this be written in a single line? Seems that the line breaks are unnecessary in this case:

```suggestion
r1, r2, r3 = _push_test_data(app, model)
```

Reply:
Inline looks better and more compact for me, but this formatting is done by ruff.